### PR TITLE
Log error responses

### PR DESCRIPTION
**Pull request by giorgiosironi**
The loggers set up on stderr and on adaptor.log ignore the extra parameter, hence error responses are not logged. By putting the responses inside the formatted message they become accessible.

Moreover, the message in http_download does not need to take into account the context of the file being an XML, as this is redundant with the message set in the `except` clause that catches it

### CODE REVIEW

File: src/adaptor.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def mkresponse(status, message, request={}, **kwargs):
         INGESTED: logging.DEBUG,
         PUBLISHED: logging.DEBUG
     }
-    LOG.log(levels[packet["status"]], "%s response", packet['status'], extra=context)
```
Comment (lsh-0) [inline src/adaptor.py]: did you mean to leave off the `extra=` in the log line?

**Comment by lsh-0** [comment]
> The loggers set up on stderr and on adaptor.log ignore the extra parameter
rather than rolling into into the message, I'd look at modifying the format lines for those logs to include the 'extra' context. good catch.

**Comment by giorgiosironi** [comment]
Feel free to amend, end result should be that on the stderr log we see the extra parameters (usually this is captured in `/var/log/upstart/bot-lax-adaptor-1.log`)

**Comment by lsh-0** [comment]
it's a little hacky because the dict passed to `extra` is merged into the Record object they're passing around behind the scenes, so what this does is make an 'extra' field available to the format line whose value is a json encoded map of all of the unidentified fields.

there's also a wee bit at the bottom of conf.py you can use to see the result for yourself.

feel free to modify the format line and merge if you're happy

**Pull request merged**